examples/pytorch/rgcn-hetero/entity_classify_mb.py

```diff
@@ -150,10 +118,11 @@ def main(args):
         if epoch > 3:
             t0 = time.time()
 
-        for i, (seeds, blocks) in enumerate(loader):
+        for i, blocks in enumerate(loader):
+            seeds = blocks[-1].dstnodes[category].data[dgl.NID]
```

Comment:
Can we return the seeds from the loader iter?

Reply:
Do you feel it's more intuitive than writing `seeds = ...`?  Should I also return the input nodes then?

Reply:
Yes, getting seed from loader is more intuitive to me

Reply:
Then what about input nodes?